### PR TITLE
Show effort spent by label

### PR DESCRIPTION
**Pull request by kdazzle**

### CODE REVIEW

**Comment by kdazzle** [comment]
Actually, I just realized that I've calculated the effort percentage based on all story points - not the completed ones like I wanted...

**Comment by kdazzle** [comment]
There. _Now_ it's ready for you @tangerilli

File: gh-sprints/web.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 from models import Sprint, IssueSnapshot, SprintCommitment, Snapshot, get_stats_for_snapshots
 from sprints import monitor_sprints
 from authorization import setup_authorization
+from statistics import LabelStatistics, LabelStatisticsCollection
```
Comment (teeberg) [inline gh-sprints/web.py]: A relative import without a dot? :-)

File: gh-sprints/statistics.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        self.total_story_points = 0
+        self.collection = collection
+
+    def add_issue(self, issue):
+        self.issues.append(issue)
+        self.total_story_points += issue.points
+
+    @property
+    def story_points_as_percentage(self):
+        percentage = (float(self.total_story_points) / float(self.collection.completed_story_points)) * 100
```
Comment (teeberg) [inline gh-sprints/statistics.py]: if you do `from __future__ import division`, you get floating point division by default and can leave it out here.

File: gh-sprints/web.py (hunk 1 above)
Comment (kdazzle) [inline gh-sprints/web.py]: I think that might actually be an absolute import

File: gh-sprints/statistics.py (hunk 2 above)
Comment (kdazzle) [inline gh-sprints/statistics.py]: ohh slick - added that!

File: gh-sprints/models.py [hunk 3]
```diff
@@ -123,6 +126,14 @@ def get_points_for_states(cls, snapshots, sprints, states=[], committed_only=Fal
             cursor = cursor.filter(IssueSnapshot.issue_id.in_(commitments))
         return cursor.scalar() or 0
 
+    def get_completed_issues(self):
+        completed_issues = []
+        for issue in self.issues:
```
Comment (tangerilli) [inline gh-sprints/models.py]: Could this just be rewritten as `return [issue for issue in self.issues if issue.state in settings.COMPLETE_STATES]`?

Comment (tangerilli) [inline gh-sprints/models.py]: Or, even better given the function you added below, `return [issue for issue in self.issues if issue.is_complete()]`

Comment (kdazzle) [inline gh-sprints/models.py]: oh yeah - that's exactly why I added that function...sure I'll do that right now.

**Pull request merged**